### ISSUE TITLE
Add code fix to use default expression for CS8313, CS8363 A default literal 'default' is not valid as a case constant / pattern.

### ISSUE THREAD

**Issue by Neme12** [dotnet/roslyn#25557]
The fix would either covert the default literal to a default expression or to another literal as described in the error message. The second option is related to and could be made as part of #25556.

Example of the error message:
```c#
class C
{
    void M()
    {
        switch (1)
        {
            case default: // A default literal 'default' is not valid as a case constant. Use another literal (e.g. '0' or 'null') as appropriate. If you intended to write the default label, use 'default:' without 'case'.
                break;
        }
    }
}
```

Expected behavior after applying code fix:
```c#
class C
{
    void M()
    {
        switch (1)
        {
            case default(int):
                break;
        }
    }
}
```

**Comment by Neme12** [comment]
I'm going to consider this fixed as part of #30359 and file another issue for a code fix to use another literal (such as 0 or null - as described in the error message).

**Comment by sharwell** [comment]
In today's design review, we felt that the presentation of the analyzer in #30360 is preferable to the one here, and that only one refactoring needs to be displayed for this situation. I'm closing it as **Won't Fix** in favor of the alternative proposal.
